.github/workflows/tests.yml

```
@@ -36,69 +44,78 @@ jobs:
             etc/eth-contracts
             etc/tests/uniswap
       - name: Install dependencies
-        run: cargo +stable make -V || cargo +stable install cargo-make
-      - name: Install wasm-opt
-        run: scripts/ci-deps/install-wasm-opt.sh
-      - name: Tests
-        run: cargo make test-flow
+        run: cargo +stable install cargo-make
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)